File: Examples.VB/Features/Formatting/CellType/AddHyperlinkCellType.vb
Namespace Features.Formatting.CellType
    Public Class AddHyperlinkCellType
        Inherits ExampleBase

        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            Dim cellType As New HyperLinkCellType With {
                .Text = "Goole",
                .LinkColor = "Blue",
                .LinkToolTip = "Search by google",
                .VisitedLinkColor = "Green",
                .Target = HyperLinkTargetType.Blank
            }

            With worksheet.Range!C5
                .CellType = cellType
                .Value = "http://www.google.com"
            End With
        End Sub
    End Class
End Namespace

File: Examples/Features/Formatting/CellType/AddHyperlinkCellType.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.CellType
{
    public class AddHyperlinkCellType : ExampleBase
    {
        public override void Execute(Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            HyperLinkCellType cellType = new HyperLinkCellType
            {
                Text = "Google",
                LinkColor = "Blue",
                LinkToolTip = "Search by google",
                VisitedLinkColor = "Green",
                Target = HyperLinkTargetType.Blank
            };

            worksheet.Range["C5"].CellType = cellType;
            worksheet.Range["C5"].Value = "http://www.google.com";
        }
    }
}
